examples/istio/main.tf

```diff
@@ -178,13 +179,39 @@ resource "helm_release" "istio_ingress" {
         }
         service = {
           annotations = {
-            "service.beta.kubernetes.io/aws-load-balancer-type"   = "nlb"
-            "service.beta.kubernetes.io/aws-load-balancer-scheme" = "internet-facing"
+            "service.beta.kubernetes.io/aws-load-balancer-type"       = "nlb"
+            "service.beta.kubernetes.io/aws-load-balancer-scheme"     = "internet-facing"
+            "service.beta.kubernetes.io/aws-load-balancer-attributes" = "load_balancing.cross_zone.enabled=true"
           }
         }
       }
     )
   ]
+  depends_on = [helm_release.istiod, helm_release.istio_base]
+}
+
+################################################################################
+# Istio Observability
+################################################################################
+
+data "http" "istio_addon" {
+  for_each = toset(local.istio_addons)
+  url      = "https://raw.githubusercontent.com/istio/istio/release-${local.istio_addons_version}/samples/addons/${each.key}.yaml"
+
+  request_headers = {
+    Accept = "text/plain"
+  }
+}
+
+data "kubectl_file_documents" "istio_addon" {
+  for_each = data.http.istio_addon
+  content  = each.value.response_body
+}
+
+resource "kubectl_manifest" "istio_addon" {
```

Comment:
we want to minimize, and eventually remove, the use of this provider since has not been updated in some time and looks like it might be abandoned https://github.com/gavinbunney/terraform-provider-kubectl/issues/247

Can we just add instructions in the document to do something like `curl <your file url> | kubectl apply -f -`

such as `curl [<your file url>](https://raw.githubusercontent.com/istio/istio/master/samples/addons/kiali.yaml) | kubectl apply -f -`